src/Admin/Controllers/OrganizationsController.cs

```diff
@@ -165,8 +182,12 @@ public async Task<IActionResult> Edit(Guid id)
         var users = await _organizationUserRepository.GetManyDetailsByOrganizationAsync(id);
         var billingInfo = await _paymentService.GetBillingAsync(organization);
         var billingSyncConnection = _globalSettings.EnableCloudCommunication ? await _organizationConnectionRepository.GetByOrganizationIdTypeAsync(id, OrganizationConnectionType.CloudBillingSync) : null;
+        var secrets = organization.UseSecretsManager ? await _secretRepository.GetManyByOrganizationIdAsync(id) : Enumerable.Empty<Secret>();
+        var projects = organization.UseSecretsManager ? await _projectRepository.GetManyByOrganizationIdAsync(id) : Enumerable.Empty<Project>();
+        var serviceAccounts = organization.UseSecretsManager ? await _serviceAccountRepository.GetManyByOrganizationIdAsync(id) : Enumerable.Empty<ServiceAccount>();
```

Comment:
As a follow-up, I think we should combine all these DB calls into 1 query and model that contains all the required information. It's now 12 separate queries just for 1 controller endpoint. What do you think? If you agree, please create a follow-up ticket and link it here.

Reply:
I agree with you. I have created the follow-up [ticket](https://bitwarden.atlassian.net/browse/AC-1569) and link it